### ISSUE TITLE
grid-column-end loses specificity battle.

### ISSUE THREAD

**Issue by christianmagill** [zurb/foundation-sites#7112]
When using grid-column-end in a basic way...

.gallery-item{
  @include grid-column(4);
  @include grid-column-end;
}

You end up with the results css selectors... and the grid-column-end selector ends up losing and not being applied.

.gallery-item:last-child:not(:first-child) {
    float: right; // generated by grid-column
}
.gallery-item:last-child {
    float: left; // generated by grid-column-end
}

**Comment by mattferderer** [comment]
The docs on Foundation's size seem to have an extra class that isn't appearing in the current version:

```CSS
.column.end:last-child, .end.columns:last-child {
    float: left; 
}
```

**Comment by gakimball** [comment]
The problem only happens when you use the mixins—the `.end` class is fine. Looking into it!

**Comment by carbinemonoxide** [comment]
Looks as if lines 70 through 72 of grid/_columns.scss takes priority over lines 43 through 48 of grid/_position.scss.

**Comment by carbinemonoxide** [comment]
The only thing I could think of to prevent this from happening was throwing an important on the float property for grid-column-end, which honestly shouldn't cause any foreseeable issues elsewhere:

https://github.com/zurb/foundation-sites/pull/7323

**Comment by gakimball** [comment]
@carbinemonoxide I'd prefer not to use `!important` if possible. It's hacky, but could you double up the `:last-child` selector to bump up the specificity?

**Comment by gakimball** [comment]
@carbinemonoxide Thanks! Merged in #7323.
